src/app/client/src/app/modules/certificate/models/cert-config-model/cert-config-model.ts

```diff
@@ -57,8 +57,25 @@ export class CertConfigModel {
 
     }
 
-    prepareCreateAssetRequest(rawFormValues) {
+    prepareCreateAssetRequest(rawFormValues, channel, images) {
         console.log(rawFormValues);
+
+    const signatoryList = [{
+       'image': _.get(images, 'SIGN1.url'),
+       'name': _.get(rawFormValues , 'authoritySignature_0'),
+     }];
+
+    if (!_.isEmpty(images['SIGN'])) {
+      signatoryList.push({
+        'image': _.get(images , 'SIGN2.url'),
+        'name': _.get(rawFormValues , 'authoritySignature_1'),
+      });
+    }
+    const issuer = {
```

Comment:
Use the "cert template layout" data.issuer values here. This should be common for the tenant irrespective of which cert template layout we select.